### DISCUSSION TITLE
battery icon - trackpad

### DISCUSSION THREAD

**Discussion by kurtper** [MultimediaLucario/Lenovo-ThinkPad-T440s#21] [Q&A] [answered]
Hi! i have a problem with battery icon, dont't show it. I have Thinkpad L440 Monterey 12.6.1

**Comment by MultimediaLucario** [comment, marked as answer]
Did you try going into settings to enable the battery icon?

**Reply by kurtper** [reply]
Yes, it’s enabled into settings but not show icon

**Reply by kurtper** [reply]
I'm sorry, the problem was that I didn't have an updated BIOS

**Reply by MultimediaLucario** [reply]
You're good.
